src/index.html

```diff
@@ -1,13 +1,43 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/indexx.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+      class="card__img"
+      src="images/imac.jpeg"
+      alt="imac"
```

Comment:
**Maybe your alt attribute is still none-informative**
![image](https://user-images.githubusercontent.com/50755505/215468760-237065c3-f88b-4610-ad49-68051bad5f06.png)